src/internal/puck/components/Header.tsx

```diff
@@ -55,10 +56,12 @@ export const customHeader = (
       <div className="header-left">
         <ToggleUIButtons />
         <ToggleEntityFields />
-        <ToggleThemeMode
-          themeModeActive={themeModeActive}
-          toggleThemeMode={setThemeModeActive}
-        />
+        {!isThemeMode && (
+          <ToggleThemeMode
```

Comment:
This is going to be removed later?

Reply:
Oh we don't want the toggleThemeMode toggle ever? If so, then yes I can remove it later

Reply:
Nope, it will be set from getting to it via the nav instead